internal/controller/nodeop_controller.go

```diff
@@ -937,6 +855,8 @@ func (r *NodeOpReconciler) createRebootPod(ctx context.Context, nodeOp *kairosio
 									rm -f "$SENTINEL_FILE"
 									echo "Attempting to patch pod..."
 									kubectl patch pod $POD_NAME -p '{"metadata":{"annotations":{"kairos.io/reboot-state":"%s"}}}' --namespace $POD_NAMESPACE || echo "kubectl patch failed"
+									echo "Giving 5 seconds to the Job Pod to exit gracefully..."
+									sleep 5
```

Comment:
i thought reboot already gave 5 seconds, so this would mean 10?

Reply:
ah no wait, this is what I was thinking about https://github.com/kairos-io/kairos-agent/blob/67e7590170680969b05e480e81ec58d84a19343c/internal/agent/hooks/lifecycle.go#L15

Reply:
:D I didn't even know about that one.